todo-list_array/script.js

```diff
@@ -0,0 +1,75 @@
+const text = document.getElementById("add-input");
+// const searchText = document.getElementById("search-input");
+const addTaskButton = document.getElementById("add-task-btn");
+const saveTaskButton = document.getElementById("save-todo-btn");
+const listBox = document.getElementById("listBox");
+const saveInd = document.getElementById("saveIndex");
+
+let todoArray = [];
+
+
+addTaskButton.addEventListener("click", (e) => {
+  if ((!text.value)== '') {
```

Comment:
Incorrect usage of syntax. Why the combination of 2 conditions here?

Either it will be: `if(text.value === ' ' ) { ... }
`
Or: `if(!text.value) { ... }`

@shubhamshukla91